addons/website_livechat/models/discuss_channel.py

```diff
@@ -32,12 +32,12 @@ def _field_store_repr(self, field_name):
                     lambda channel: Store.One(
                         channel.livechat_visitor_id,
```

Comment:
Note: I'm not sure if I created the task yet, but we want to eventually rename "visitor" in thread (js) to `livechat_visitor_id` to match python.

This PR might be the perfect opportunity for doing it too, so we can see and double check all fields usage in the same diff.